syntax = "proto2";

message CurrentStateRequest {
}
